scripts/replace-external-deps-with-cdn.mjs

```diff
@@ -0,0 +1,63 @@
+import {exec} from 'child_process';
```

Comment:
This script gets the current version of headless/atomic and uses that when replacing references to @coveo/headless and @coveo/atomic with CDN URLs. Once we add bueno to the CDN we need to have this script handle it too